robolectric-annotations/src/main/java/org/robolectric/annotation/Config.java

```diff
@@ -25,6 +25,7 @@
   String DEFAULT = "--default";
   String DEFAULT_RES_FOLDER = "res";
   String DEFAULT_ASSET_FOLDER = "assets";
+  String dEFAULT_RENDERING_ENABLED = "false";
```

Comment:
Should it default to true? We don't want to enable layout lib by default yet right?